datasets/titanic/metadata.json

```diff
@@ -116,15 +116,15 @@
             "name": "gender",
             "description": "Gender of passenger (0: Male, 1: Female)",
             "@type": "ml:Field",
-            "dataType": "sc:Integer",
+            "dataType": "sc:Text",
```

Comment:
Note to the reviewer: for now we haven't implemented the join with the gender enums, so it's a `Text`, not an `Integer`.

Reply:
I would prefer we support casting right away, ie: if the read type (string) is compatible with specified type (number), then just casting to specified type. I don't think this needs to wait for join.

 Alternatively, could you add a README.md in directory with a todo so people who read the example aren't confused?

Reply:
Sorry my mistake: it's "female" in dataset, so indeed Text.
Ignore my previous comment, thanks!